### DISCUSSION TITLE
Strange fan speed when setting Turbo mode.

### DISCUSSION THREAD

**Discussion by cheonmux** [seerge/g-helper#257] [Q&A]
Hello.

First of all, thank you so much for developing such a useful program.

I'm using a G15 (2022) 503 model, and the plugin works great.

However, I have a problem that when I set the turbo mode, the fan speed is activated to the maximum.

[Screenshot](https://imgsh.net/a/JpeO2BY.jpg)

The CPU temperature in the screenshot is 74 degrees, and according to the fan curve, it should be running at 4400 RPM (about 70%). 

However, in my case, it always runs at 97%. Is it normal for the fan to run at full speed when set to turbo mode?

**Comment by seerge** [comment]
@cheonmux this is discussions section, not [issues ](https://github.com/seerge/g-helper/issues).
You should post a log from your app there if you want it to be checked somehow. However app doesn't control your fans (in realtime) your bios does https://github.com/seerge/g-helper#how-does-g-helper-control-my-fan-speeds
